tests/utils/dataset.py

```diff
@@ -315,10 +318,11 @@ def get_test_dataset_with_image_patches(
     )
 
 
-def get_model_test_dataset(config_only: bool = False):
+def get_model_test_dataset(config_only: bool = False, use_loss_masking: bool = False):
     return _get_test_dataset(
         DATASET_CACHE / "model_dataset",
         seed=1234,
+        max_loss_masking_spans=5 if use_loss_masking else 0,
```

Comment:
This won't work, the dataset is cached so the argument needs to be constant. `max_loss_masking_spans=5` should work